sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateContentType.cs

```diff
@@ -7,7 +7,7 @@
 namespace Azure.Security.KeyVault.Certificates
 {
     /// <summary>
-    /// Content type of the certificate when downloaded from getSecret.
+    /// Content type of the certificate when the managed secret is downloaded using a <c>SecretClient</c>.
```

Comment:
nit: Without the `cref`, is it worth wrapping this in a code tag?

Reply:
IMO, makes it more obvious it's a bit o' code. Obviously don't want a package dependency - especially for a comment - so I felt this was more obvious and a compromise.